starter-cli/src/main/java/io/micronaut/starter/cli/command/CreateCommand.java

```diff
@@ -44,15 +45,15 @@
 
     @ReflectiveAccess
     @CommandLine.Option(names = {"-l", "--lang"}, paramLabel = "LANG", description = "Which language to use. Possible values: ${COMPLETION-CANDIDATES}.", completionCandidates = LanguageCandidates.class, converter = LanguageConverter.class)
-    Language lang;
+    Language lang = Language.DEFAULT_OPTION;
```

Comment:
We can't set a default language here because the language can be inferred based on the selected features

Reply:
The build features and test features have the knowledge of what is the default currently and they apply themselves based on the build / test options not being set